### PR TITLE
PP-452: Remove PTL code related to interface that doesnt exist in PBS…

### PR DESCRIPTION
**Pull request by saritakh**
#### Issue-ID

* **[PP-452](https://pbspro.atlassian.net/browse/PP-452)**

#### Problem description
* PTL library code contains a command line argument that doesnt exist in PBS anymore

#### Cause / Analysis
* Remove PTL code related to interface that doesnt exist in PBS anymore

#### Solution description
* Removal PTL code related to interface that doesnt exist in PBS anymore

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [ ] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

… anymore

### CODE REVIEW

**Comment by hirenvadalia** [comment]
Changes looks good. I sign-off.

File: test/fw/ptl/lib/pbs_testlib.py [hunk 1]
```diff
@@ -4235,8 +4235,7 @@ class Comm(PBSService):
     logger = logging.getLogger(__name__)
     logprefix = 'pbs_comm: '
 
-    conf_to_cmd_map = {'PBS_COMM_NAME': '-n', 'PBS_COMM_ROUTERS': '-r',
-                       'PBS_COMM_THREADS': '-t'}
+    conf_to_cmd_map = {'PBS_COMM_ROUTERS': '-r', 'PBS_COMM_THREADS': '-t'}
```
Comment (bhroam) [inline test/fw/ptl/lib/pbs_testlib.py]: I looked up the command line options to pbs_comm and there is a -N option to run in standalone mode.  Is this something we need to care about?

**Comment by saritakh** [comment]
@bhroam As of now I dont think there is any requirement to run the PBS daemons on standalone mode from PTL. I think it can be addressed if the requirement arrives.

**Comment by bhroam** [comment]
OK, sounds good to me.  I just saw it was an option which PTL didn't know about.  If you're doing it on purpose, then it is fine.  

Please bring your branch up to current and I'll merge it in.

**Comment by saritakh** [comment]
Updated my branch. Kindly take this forward. 
Thanks.

**Pull request merged**